.travis.yml

```diff
@@ -60,46 +60,73 @@ jobs:
 
         - &STANDARD_TEST_JOB
             stage: Fast Test
-            php: 7.0
+            php: '7.2'
             install:
                 # Composer: enforce given Symfony components version
                 - if [ "$SYMFONY_VERSION" != "" ]; then composer global show symfony/flex -q || travis_retry composer global require $DEFAULT_COMPOSER_FLAGS symfony/flex; fi
                 - if [ "$SYMFONY_VERSION" != "" ]; then composer config extra.symfony.require $SYMFONY_VERSION || true; fi
 
-                - travis_retry composer update $DEFAULT_COMPOSER_FLAGS $COMPOSER_FLAGS
+                - travis_retry composer update --working-dir=./dev-tools $DEFAULT_COMPOSER_FLAGS $COMPOSER_FLAGS
                 - composer info -D | sort
             script:
-                - vendor/bin/phpunit || travis_terminate 1
+                - ./dev-tools/vendor/bin/phpunit || travis_terminate 1
                 - PHP_CS_FIXER_FUTURE_MODE=1 php php-cs-fixer --diff --dry-run -v fix
 
         -
             <<: *STANDARD_TEST_JOB
             stage: Test
-            php: 5.6
+            php: '5.6'
             env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
+            before_install:
+                # turn off XDebug
```

Comment:
See https://twitter.com/derickr/status/1184465999971700736.

😉

Reply:
@localheinz fixed, now you can safely review the rest of PR ;)